lib/ex_schedule.ex

```diff
@@ -100,7 +100,7 @@ defmodule ExSchedule do
 
       @doc false
       def init(opts) do
-        schedules() |> Enum.map(&child_spec/1) |> supervise(strategy: :one_for_one)
+        schedules() |> start() |> Supervisor.init(strategy: :rest_for_one)
```

Comment:
[question] Why the change to `rest_for_one`? 
With `rest_for_one`:

> if a child process terminates, the terminated child process and the rest of the children started after it, are terminated and restarted.

But there should be no interdependencies between the workers, so `one_for_one` seems more appropriate.